--- a/test-src/com/dumbster/smtp/BindProblemTest.java
+++ b/test-src/com/dumbster/smtp/BindProblemTest.java
@@ -24,6 +24,7 @@
  */
 public class BindProblemTest extends TestCase {
 
+	private static final int SMTP_PORT = 1081;
   private SimpleSmtpServer server;
 
   /**
@@ -31,7 +32,7 @@
    */
   protected void setUp() throws Exception {
     super.setUp();
-    server = SimpleSmtpServer.start();
+    server = SimpleSmtpServer.start(SMTP_PORT);
   }
 
   /**
